apps/fluent-tester/src/RNTester/Consts.ts

```diff
@@ -0,0 +1,51 @@
+// Button Test Page
+export const HOMEPAGE_BUTTON_BUTTON = 'Homepage_Button_Button';
```

Comment:
One small thing worth considering - maybe these constants should be moved to their respective test pages. That feels a little cleaner to me, but if there's a good reason it's like this please share.

Reply:
I followed RNW's structure of one constants file. I have no preference though. I will move these to their respective test pages

Reply:
Actually, I just remembered a reason. I'm unsure if the test pages will be the exact same across platform. If I have to make three different test pages for each component for iOS, Windows, Android (for example), would the constants go in each one? In that case, I think it would be better for one consts file. What do you think?

Reply:
So this is where directory structure is important. A part of me hopes that the split on platform is a leaf level directory and then you can share the consts across the three different versions in the parent directory of the three test pages. I'm not actually sure what the plan there is.

Reply:
I am creating a separate task to track this. I will make a post in our teams chat about possible structure.